barretenberg/cpp/src/barretenberg/relations/utils.hpp

```diff
@@ -114,67 +113,6 @@ template <typename Flavor> class RelationUtils {
         }
     }
```

Comment:
Code in the two function below was getting more and more complicated in the interest of being shared between PG and sumcheck so decided to separate it again

Reply:
Fair enough